### ISSUE TITLE
WPS237: Found a too complex `f` string - false positive?

### ISSUE THREAD

**Issue by luminoso** [wemake-services/wemake-python-styleguide#2491]
### What's wrong

WPS237 signals on not-complex pydantic fstrings.

Example:

```python
from enum import Enum

from pydantic import BaseModel


class Fuel(Enum):
    disel = "disel"
    gpl = "gpl"
    electric = "electric"


class Payload(BaseModel):
    car_fuel: Fuel
    car_name: str


def wemake_python_sg(request_car: Payload):
    print(f"{request_car.car_type}")
    print(f"{request_car.car_fuel.disel}")
```

Where:
`print(f"{request_car.car_type}")` doesn't signal WPS237, but 
`print(f"{request_car.car_fuel.disel}")` does

### How it should be

Both strings are well-defined and not complex. Sounds like a false positive.

### Flake8 version and plugins
```json
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.10.6",
    "system": "Linux"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "dlint",
      "version": "0.13.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-bandit",
      "version": "3.0.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-broken-line",
      "version": "0.4.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-bugbear",
      "version": "22.9.11"
    },
    {
      "is_local": false,
      "plugin": "flake8-comprehensions",
      "version": "3.10.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-darglint",
      "version": "1.8.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-debugger",
      "version": "4.1.2"
    },
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.6.0, pydocstyle: 6.1.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-eradicate",
      "version": "1.3.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-string-format",
      "version": "0.3.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_commas",
      "version": "2.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_isort",
      "version": "4.2.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_quotes",
      "version": "3.3.1"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "naming",
      "version": "0.12.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.8.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.4.0"
    },
    {
      "is_local": false,
      "plugin": "rst-docstrings",
      "version": "0.2.7"
    },
    {
      "is_local": false,
      "plugin": "wemake_python_styleguide",
      "version": "0.16.1"
    }
  ],
  "version": "4.0.1"
}
```

### pip information

(not applicable)

### OS information

LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: Fedora
Description:    Fedora release 36 (Thirty Six)
Release:        36
Codename:       ThirtySix

**Comment by sobolevn** [comment]
No, this is intended. We don't allow too deep attribute access in this rule. Only one level is allowed. This is why the first string does not raise.

**Comment by luminoso** [comment]
Gotcha.
What's the vision for readable and less complex code in this case?

Should it look like this? Feels redundant. Any other suggestion how to make it readable and compliant? 

```python
def wemake_python_sg(request_car: Payload):
    print(f"{request_car.car_type}")
    car_fuel = request_car.car_fuel
    print(f"{car_fuel.disel}")
```

**Comment by sobolevn** [comment]
I don't ever use `f` strings at all :)
So, I would suggest:

```python
def wemake_python_sg(request_car: Payload):
    print("Whatever {0}".format(request_car.car_fuel.diesel))
```

**Comment by gamer191** [comment]
I found another possible false positive:
An f-string with no placeholders will trigger WPS237 (as well as F541)

Whilst it's never gonna actually be an issue (because of F541), `print(f'hi')` technically shouldn't trigger WPS237
